### PR TITLE
Bug 1217758 - Add a dropmarker to filters button

### PR DESCRIPTION
**Pull request by MekliCZ**

### CODE REVIEW

**Comment by mathjazz** [comment]
Thanks for the PR. The code looks ok.

I'm not sure about the appearance though. The idea sounds good, but it doesn't look very slick. ;) And it's hard to find the position of the caret that would work with al the icons we use.

What do you think? Maybe center align the caret at the bottom of the icon (`bottom: 2px;` and `left: 19px;`)?

File: pontoon/base/static/css/translate.css [hunk 1]
```diff
@@ -696,7 +707,7 @@ body > header aside p {
   color: #AAAAAA;
   display: none;
   font-size: 16px;
-  left: 43px;
+  left: 53px;
```
Comment (mathjazz) [inline pontoon/base/static/css/translate.css]: That should remain `43px`. `42px` looks even better.

Comment (MekliCZ) [inline pontoon/base/static/css/translate.css]: @mathjazz It looks better, but when there is caret, it looks really bad. But i can hide caret when menu is opened, what do you think?

**Comment by MekliCZ** [comment]
@mathjazz I changed caret position to bottom of the icons, but I have to use decimals in px values for the best look on my Retina display. `18px` is not enough, `19px` is too big. I'm not sure that decimals in px are the best way, what do you think?

**Comment by mathjazz** [comment]
What if you change the icon font-size?

**Comment by MekliCZ** [comment]
I can't find the right combination.

**Comment by mathjazz** [comment]
Could you paste a screenshot demonstrating the issue?

**Comment by MekliCZ** [comment]
Caret is slightly shifted to the left.
snimek obrazovky 2016-04-11 v 22 41 11

**Comment by mathjazz** [comment]
Doesn't look too bad. Or does it in real app? Could you commit?

**Comment by MekliCZ** [comment]
I see it in app. Maybe it's my personal view, but I see that few pixels are wrong here.

**Comment by mathjazz** [comment]
If you think it's worse than what's currently in production, we can also wontfix the bug. :)

**Comment by MekliCZ** [comment]
Great idea, I can find another bug to fix as my first.

**Comment by mathjazz** [comment]
Thanks for giving it a try!

**Pull request closed without merging**